### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by buhongw7583c**
Add the procedures running the deployment inside a docker container.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -70,6 +70,24 @@ Please report any problems you face under issues!
 - [Node.js & npm](https://nodejs.org/en/download/) >= 8.5
 - [LUIS CLI](https://github.com/Microsoft/botbuilder-tools/tree/master/packages/LUIS) >=2.6.2 
 - Be logged into Azure CLI and having Subscription Level Owner or Contributor rights on the "isDefault" marked subscription
+- You can also running the deployment in the docker container following the steps below:
+     1. Clone the repo to the local driver;
```
Comment (h2floh) [inline README.md]:
```suggestion
     1. Clone the repo to your local drive
```

File: README.md [hunk 2]
```diff
@@ -70,6 +70,24 @@ Please report any problems you face under issues!
...
+     2. Install the image of  the h2floh/geobotagent which has the prerequisite software installed:
```
Comment (h2floh) [inline README.md]:
```suggestion
     2. Pull the docker image `h2floh/geobotagent` which has the prerequisite software installed: 
```

File: README.md [hunk 3]
```diff
@@ -70,6 +70,24 @@ Please report any problems you face under issues!
...
 - Be logged into Azure CLI and having Subscription Level Owner or Contributor rights on the "isDefault" marked subscription
+- You can also running the deployment in the docker container following the steps below:
+     1. Clone the repo to the local driver;
+     2. Install the image of  the h2floh/geobotagent which has the prerequisite software installed: 
+
+         `
+         docker pull  h2floh/geobotagent
+         `
+
+     3. In your docker setting, ensure the driver contains the repo is shared and would be available to your containers created in next step.
```
Comment (h2floh) [inline README.md]:
```suggestion
     3. In your docker setting, ensure the drive, which contains the repo, is shared and would be available to your containers created in next step. 
```

File: README.md [hunk 4]
```diff
@@ -70,6 +70,24 @@ Please report any problems you face under issues!
...
+         `
+
+     3. In your docker setting, ensure the driver contains the repo is shared and would be available to your containers created in next step. 
+     4. Execute the command on your terminial: 
+
+         `
+         docker run -it --name <botname> --mount type=bind,source="$(pwd)"/GeoDistributedAzureBot,target=/<foldername> h2floh/geobotagent:latest bin/bash
+         `
+
+         The command is to [mount](https://docs.docker.com/storage/bind-mounts/) an existing folder to the docker container and start the container from bash terminal. The mounted files will be located in the target place specified by the foldername.
```
Comment (h2floh) [inline README.md]:
```suggestion
         The command will [mount](https://docs.docker.com/storage/bind-mounts/) the existing folder to the docker container and start the container from bash terminal. The mounted files will be located in the target place specified by `<foldername>`. 
```

File: README.md [hunk 5]
```diff
@@ -70,6 +70,24 @@ Please report any problems you face under issues!
...
+
+         `
+         docker pull  h2floh/geobotagent
+         `
+
+     3. In your docker setting, ensure the driver contains the repo is shared and would be available to your containers created in next step. 
+     4. Execute the command on your terminial: 
+
+         `
+         docker run -it --name <botname> --mount type=bind,source="$(pwd)"/GeoDistributedAzureBot,target=/<foldername> h2floh/geobotagent:latest bin/bash
```
Comment (h2floh) [inline README.md]: Doesn't the command have to be `pwsh`?

**Review by h2floh — COMMENTED** [review]
Hi @buhongw7583c ,

thanks for contributing! 👍  Some suggestions for grammar and wording, hope you don't mind.

Does it work if you execute the container in `bash` mode? Shouldn't it be `pwsh` ? Never tried to execute the scripts from bash.

File: README.md (hunk 5 above)
Comment (angie4u) [reply README.md]: I also never tried using bash! 😏

**Review by angie4u — APPROVED** [review]

**Review by h2floh — COMMENTED** [review]
Thanks @buhongw7583c ! 
LGTM 👍

**Pull request merged**